cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -229,6 +229,13 @@ function Fetch-KubeEnv {
   # The type of kube_env is a powershell String.
   $kube_env = Get-InstanceMetadataAttribute 'kube-env'
   $kube_env_table = ConvertFrom_Yaml_KubeEnv ${kube_env}
+
+  Write-Host "Logging kube-env key-value pairs except CERT and KEY values"
+  foreach ($entry in $kube_env_table.GetEnumerator()) {
+    if ((-not ($entry.Name.contains("CERT"))) -and (-not ($entry.Name.contains("KEY")))) {
+      Write-Host "$($entry.Name): $($entry.Value)"
```

Comment:
Write_Output?

Reply:
haven't seen that usage yet. I have seen Write-Host and Log-Output

Reply:
Yes use Log-Output.